src/renderers/dom/client/utils/getEventTarget.js

```diff
@@ -20,6 +20,12 @@
  */
 function getEventTarget(nativeEvent) {
   var target = nativeEvent.target || nativeEvent.srcElement || window;
+
+  // Normalize SVG <use> element events #4963
+  if (target.correspondingUseElement && !target.nodeType) {
```

Comment:
I've looked over and retested everything. `!target.nodeType` is a relic from my exploration during early implementation that only partially solved the issues on IE, whereas `correspondingUseElement` properly solves it. I'll remove it and rebase onto the current commit so there's no noise to merge.